web/frontend/src/api/navigation.ts

```diff
@@ -53,6 +53,7 @@ export const getObstacles = async (navClient: NavigationClient): Promise<Obstacl
 
           return {
             type: 'sphere',
+            // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
If the intention is to fix these in the future, I think a `warn` in the ESlint config might be a better way to go so we don't forget about them and/or to encourage folks to fix them incrementally

Reply:
Yeah, after thinking about this more I agree, fixed!